From 2f571829d7f36165d690386d208e92b62a818995 Mon Sep 17 00:00:00 2001
From: Graham Ollis <plicease@cpan.org>
Date: Thu, 23 Mar 2017 06:40:21 -0400
Subject: [PATCH] Add '.' to @INC in Makefile.PL

Perl 5.25.11 no longer has '.' in the @INC path by default.
---
 Makefile.PL | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.PL b/Makefile.PL
index 4db553e..eebd90f 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -3,6 +3,7 @@
 use strict;
 use warnings;
 use 5.005;
+use lib '.';
 use inc::Module::Install;
 
 name		'Authen-SASL';
